Automattic/wp-calypso
# Adding educational card with link to videos UI modal

#### Changes proposed in this Pull Request

This PR adds a new card to all current "My Home" layouts with a link to a modal displaying the new videos UI.

This PR will be the central point of reference for our CFT for this feature, and should not be merged until the feature is fully approved. (It will probably require further updates as more modifications are made to the UI component being loaded in the modal.)

#### Testing instructions



* Check out D69391-code to your sandbox and sandbox the API. (Not necessary if D69391 has already been merged.)
* Check out this PR locally.
* Verify that the Calypso homepage now displays a card in the "education" carousel as follows:

![image](https://user-images.githubusercontent.com/13437011/140367907-cafe66ed-afd2-4c9a-ac1c-88218ecdce2a.png)

* Verify that clicking on the link in the card opens a modal popup displaying the videos UI:

![image](https://user-images.githubusercontent.com/13437011/140368247-fe241a0b-8308-4d10-a706-ce11404ec72e.png)



Related to #57326
